helm/amazon-ec2-metadata-mock/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: amazon-ec2-metadata-mock
 description: A Helm chart for the Amazon EC2 Metadata Mock
-version: 0.5.7
+version: 0.5.8
```

Comment:
may be we should update the minor version since this is feature addition?

Reply:
Also, in order to release this version of the chart to be able to use it in NTH, we will need to release AEMM. 
So, you might want to factor that in or update minor version and then update the new version;s patch version again for release.

Reply:
update version to 1.0.1 (same as app version)